spec/definitions.yaml

```diff
@@ -962,6 +962,34 @@ definitions:
             description: |
               Name of the availability zone the master node is placed in
 
+  V5GetClusterStatusResponse:
+    type: object
+    description: Cluster status information
+    properties:
+      conditions:
+        type: array
+        description: List of conditions the cluster has gone through
+        items:
+          type: object
+          properties:
+            last_transition_time:
+              type: string
+              description: Date and time when the cluster transitioned into this condition
+            type:
```

Comment:
When the list is called `conditions`, would it make sense to call this here `condition` instead? Below we have the list called `versions` and the item's field called `version`. That is consistent and comprehensible. I always struggled with `type` and now that we cleanup and revamp the status structures with new types, I would like to have more feedback about this and maybe make it better. I would then also like to adapt internally accordingly.

Reply:
I think you are right, `condition` would be better than `type`.

Reply:
Let me know what the consensus is. Then I am happy to provide PRs for the internal changes in the operators and `apiextensions`.

Reply:
The consensus is `condition`. See https://github.com/giantswarm/api-spec/pull/130/commits/dc7a27c166ee9f3d0f0f6fb8ab111c0589bc19bf